### PR TITLE
Link to docs from the Concurrent Mode feature comparison table

### PR DESCRIPTION
**Pull request by keyz**
Think this will make it easier for people to reference the features

Screen Shot 2019-11-09 at 2 06 23 AM

### CODE REVIEW

**Comment by keyz** [comment]
https://deploy-preview-2550--reactjs.netlify.com/docs/concurrent-mode-adoption.html#feature-comparison

File: content/docs/concurrent-mode-adoption.md [hunk 1]
```diff
@@ -107,21 +107,21 @@ You can think of the Blocking Mode as a "gracefully degraded" version of the Con
 
 |   |Legacy Mode  |Blocking Mode  |Concurrent Mode  |
 |---  |---  |---  |---  |
-|String Refs  |✅  |🚫**  |🚫**  |
-|Legacy Context |✅  |🚫**  |🚫**  |
-|findDOMNode  |✅  |🚫**  |🚫**  |
-|Suspense |✅  |✅  |✅  |
-|SuspenseList |🚫  |✅  |✅  |
+|[String Refs](/docs/strict-mode.html#warning-about-legacy-string-ref-api-usage)  |✅  |🚫**  |🚫**  |
+|[Legacy Context](/docs/context.html#legacy-api) |✅  |🚫**  |🚫**  |
```
Comment (lex111) [inline content/docs/concurrent-mode-adoption.md]:
```suggestion
|[Legacy Context](/docs/legacy-context.html) |✅  |🚫**  |🚫**  |
```

File: content/docs/concurrent-mode-adoption.md [hunk 2]
```diff
@@ -107,21 +107,21 @@ You can think of the Blocking Mode as a "gracefully degraded" version of the Con
 
 |   |Legacy Mode  |Blocking Mode  |Concurrent Mode  |
 |---  |---  |---  |---  |
-|String Refs  |✅  |🚫**  |🚫**  |
-|Legacy Context |✅  |🚫**  |🚫**  |
-|findDOMNode  |✅  |🚫**  |🚫**  |
-|Suspense |✅  |✅  |✅  |
-|SuspenseList |🚫  |✅  |✅  |
+|[String Refs](/docs/strict-mode.html#warning-about-legacy-string-ref-api-usage)  |✅  |🚫**  |🚫**  |
```
Comment (lex111) [inline content/docs/concurrent-mode-adoption.md]:
```suggestion
|[String Refs](/docs/refs-and-the-dom.html#legacy-api-string-refs)  |✅  |🚫**  |🚫**  |
```

**Comment by keyz** [comment]
@lex111 thanks for the feedback!

**Comment by lex111** [comment]
@keyz seems legit, thank you!

**Pull request merged**